packages/fiber/src/react/reconciler.ts

```diff
@@ -0,0 +1,216 @@
+import {
```

Comment:
The reconciler was the React-specific part of the `createRenderer`, splitting it into two parts, where the `core` exposes a renderer and the `ReactReconciler` operates using the renderer.